Installer.AdminApi/install.ps1

```diff
@@ -86,15 +86,15 @@ $adminApiSource = "$packageSource/AdminApi"
 $p = @{
     ToolsPath = "C:/temp/tools"
     DbConnectionInfo = $dbConnectionInfo
-    OdsApiUrl = ""
+    OdsApiVersion = "6.1"
     PackageVersion = '1.3.1.0'
```

Comment:
@simpat-adam 
Please remove "6.1" and leave it as empty string. User will be forced to provide correct value.